test/e2e/runner/main.go

```diff
@@ -187,20 +191,10 @@ func NewCLI() *CLI {
 	})
 
 	cli.root.AddCommand(&cobra.Command{
-		Use:   "load [multiplier]",
-		Args:  cobra.MaximumNArgs(1),
+		Use:   "load",
 		Short: "Generates transaction load until the command is canceled",
 		RunE: func(cmd *cobra.Command, args []string) (err error) {
-			m := 1
-
-			if len(args) == 1 {
-				m, err = strconv.Atoi(args[0])
-				if err != nil {
-					return err
-				}
-			}
-
-			return Load(context.Background(), cli.testnet, m)
+			return Load(context.Background(), cli.testnet)
```

Comment:
What's your reason for getting rid of the multiplier thing here?

Reply:
I got rid of it in other places, in favor of a simpler jitter method for controlling the amount of load. I found the multiplier hard to reason about, and we were seeing some transaction load related errors that I think were spurious, so it seemed like a reasonable simplification.